Formula/s/slepc-complex.rb

```diff
@@ -0,0 +1,40 @@
+class SlepcComplex < Formula
+  desc "Scalable Library for Eigenvalue Problem Computations (complex)"
+  homepage "https://slepc.upv.es"
+  url "https://slepc.upv.es/download/distrib/slepc-3.22.2.tar.gz"
+  sha256 "b60e58b2fa5eb7db05ce5e3a585811b43b1cc7cf89c32266e37b05f0cefd8899"
+  license "BSD-2-Clause"
+
+  livecheck do
+    formula "slepc"
+  end
+
+  depends_on "petsc-complex"
+
+  uses_from_macos "python" => :build
+
+  conflicts_with "slepc", because: "slepc must be installed with either real or complex support, not both"
+
+  def install
+    ENV["PETSC_DIR"] = Formula["petsc-complex"].prefix.realpath
+    ENV["SLEPC_DIR"] = Dir.pwd
```

Comment:
What is this used for? Because that looks like something that might cause the Linux failure if it's looking for libraries in the tmp build directory

Reply:
The environment variables are needed for building the library, but after `make install` they are not necessary.